src/main/java/core/basesyntax/JavaStreamApi.java

```diff
@@ -75,7 +87,11 @@ public Double averageSumOdd(List<Integer> numbers) {
      **/
     public List<People> workablePeople(int fromAge, int femaleToAge,
                                        int maleToAge, List<People> peopleList) {
-        return Collections.emptyList();
+        return peopleList.stream()
+                .filter(p -> p.getAge() >= fromAge)
+                .filter(p -> p.getSex() == People.Sex.MAN && p.getAge() <= maleToAge
```

Comment:
though Bohdan showed two filters, I was told not to have two - combine into one filter